src/main/js/controls/Gantt/Gantt.js

```diff
@@ -287,20 +290,22 @@ class Gantt extends Construct {
      * The content serves as a 1to1 relationship. For rendering
      * multiple data sets respective number of content needs to be provided.
      *
-     * @param {object} content - Gantt content to be removed
+     * @param {object|Array} content - Gantt content to be removed
      * @returns {Gantt} - Gantt instance
      */
     unloadContent(content) {
-        const index = this.tracks.indexOf(content.key);
-        if (index < 0) {
-            throw new Error(errors.THROW_MSG_INVALID_OBJECT_PROVIDED);
-        }
-        this.trackConfig[index].unload(this);
+        contentLoadHandler(content, (i) => {
```

Comment:
I am wondering if it will be safer for us to create a `contentUnloadHandler` in this case.

Reply:
Agree. Do you prefer another file (unloadUtil) for this or just add to loadUtil.js

Reply:
Ah good point. How about having the function `contentUnloadHandler` in the same file but rename the file to `constructUtils.js` (loadContent and unloadContent are Construct functions)